robyn/__init__.py

```diff
@@ -129,6 +135,12 @@ def patch(self, endpoint):
         :param endpoint [str]: [endpoint to server the route]
         """
         def inner(handler):
+            sig = inspect.signature(handler)
+            params = len(sig.parameters)
+            if params != 1:
```

Comment:
On further thought, this should not be a mandate.
One should be able to create a route without worrying about the params.